src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine{

```

Comment:
There should not be a redundant empty line after the class declaration. Please remove the empty line after the `Truck` class declaration to follow Java code conventions.